.env

```diff
@@ -39,4 +39,6 @@ RABBITMQ_PORT=5672
 # LDAP USER MAPPING
 LDAP_USER_GROUP=
 
+CSRF_SECRET="P*3NGEEaJV3yUGDJA9428EQRg!ad"
```

Comment:
reminder: this should be changed by the user for security reasons. Remember to include this in the docs.